openlibrary/templates/books/author-autocomplete.html

```diff
@@ -44,6 +44,7 @@
         $:render_author(name_path, dict_path, loop.index0, author)
 </div>
 <script type="text/javascript">
+document.getElementById('author-$i').disabled = false;
```

Comment:
Can you move this line underneath `window.q.push( function () {` ?
This will make sure that this triggers only when the field is setup.

Reply:
Also $i is not defined here!

$i is only available inside the jsdef command.

A better approach would be to use the selector `.author-autocomplete` e.g. `$('.author-autocomplete').prop('disabled', false );` for example.